src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star :last-child {
+  margin-left: 0;
+}
```

Comment:
Avoid styling using tag names as selectors. It's recommended to use classes instead to ensure that the styles are not too tightly coupled with the HTML structure and to prevent unintended side effects in other parts of the application. For example, instead of `.stars__star :last-child`, consider using a class like `.stars__star--last`.